pyproject.toml

```diff
@@ -28,6 +28,17 @@ pypi-latest = ">=0.1.1"
 muon = ">=0.1.2"
 requests = ">=2.27.1"
 ipywidgets = ">=7.6.5"
+sphinx-gallery = "^0.10.1"
```

Comment:
Now the sphinx dependencies are also in the normal dependencies. They should only be in the dev dependencies (as they were!). I could tell you why the nox session didn't quite behave as you expected it to, but it doesn't matter anymore.

Please remove them again from here and update the lock file.

Reply:
Ah Im sorry! I did not realise those differences. I just noticed that I had to install all of those things to run nox --session "build-docs" to build the docs but I think I know a different way now anyway.

Reply:
No problem :)

Reply:
I kept the switchlang and sklearn-misc but removed everything sphinx.